### PR TITLE
WEBAPP-36

### PR DESCRIPTION
**Pull request by Schedulaar**
Add link to disclaimer to footer

### CODE REVIEW

File: locales/ar/common.json [hunk 1]
```diff
@@ -1 +1,3 @@
-{}
+{
+  "ImprintAndContact": "هيئة التحرير وجهة الاتصال"
```
Comment (maxammann) [inline locales/ar/common.json]: We should stick to camelCase here

File: locales/de/common.json [hunk 2]
```diff
@@ -1 +1,3 @@
-{}
+{
+  "ImprintAndContact": "Impressum und Kontakt"
```
Comment (maxammann) [inline locales/de/common.json]: see above and you should create a new language file Footer.json, see Search.json

Comment (Schedulaar) [inline locales/de/common.json]: I think maintaining all these language files seperately is more work than just one file per language. To be considered: There are about 10 languages to support ~ if we have ten different sections like footer, header,... we got 100 files which is very messy. Or do we have some other tools to manage these files?

Comment (maxammann) [inline locales/de/common.json]: true, i18next-resource-store-loader is used to generate the translations.
The impementation of the translations is in i18n. Different ns (namespaces) mean different files. We could merge all C
component translations in one json file

**Pull request merged**